Package.swift

```diff
@@ -0,0 +1,8 @@
+import PackageDescription
+
+let package = Package(
+    name: "SQLiteSDK",
+    providers: [
+        .Apt("libsqlite3-dev")
```

Comment:
OK I understand - I guess that the `-dev` suffix just means that it comes with `sqlite3.h`.

The question of the version that comes preinstalled on macOS/iOS is important in GRDB, because some features are protected with @available attributes. But I'm sure SPM supports them correctly.

For Linux, we'll need to talk about the SQLite version with more details. But we're not there yet :-)